static/app/views/dashboards/manage/dashboardTable.tsx

```diff
@@ -0,0 +1,232 @@
+import {Fragment} from 'react';
+import styled from '@emotion/styled';
+import type {Location} from 'history';
+
+import {
+  createDashboard,
+  deleteDashboard,
+  fetchDashboard,
+} from 'sentry/actionCreators/dashboards';
+import {addErrorMessage, addSuccessMessage} from 'sentry/actionCreators/indicator';
+import type {Client} from 'sentry/api';
+import {ActivityAvatar} from 'sentry/components/activity/item/avatar';
+import {Button} from 'sentry/components/button';
+import {openConfirmModal} from 'sentry/components/confirm';
+import EmptyStateWarning from 'sentry/components/emptyStateWarning';
+import GridEditable, {
+  COL_WIDTH_UNDEFINED,
+  type GridColumnOrder,
+} from 'sentry/components/gridEditable';
+import Link from 'sentry/components/links/link';
+import TimeSince from 'sentry/components/timeSince';
+import {IconCopy, IconDelete} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import type {Organization} from 'sentry/types/organization';
+import {trackAnalytics} from 'sentry/utils/analytics';
+import withApi from 'sentry/utils/withApi';
+import type {DashboardListItem} from 'sentry/views/dashboards/types';
+
+import {cloneDashboard} from '../utils';
+
+type Props = {
+  api: Client;
+  dashboards: DashboardListItem[] | undefined;
+  location: Location;
+  onDashboardsChange: () => void;
+  organization: Organization;
+  isLoading?: boolean;
+};
+
+enum Keys {
+  NAME = 'title',
+  WIDGETS = 'widgetDisplay',
+  OWNER = 'createdBy',
+  CREATED = 'dateCreated',
+}
+
+function DashboardTable({
+  api,
+  organization,
+  location,
+  dashboards,
+  onDashboardsChange,
+  isLoading,
+}: Props) {
+  const columnOrder = [
+    {key: Keys.NAME, name: t('Name'), width: COL_WIDTH_UNDEFINED},
+    {key: Keys.WIDGETS, name: t('Widgets'), width: COL_WIDTH_UNDEFINED},
+    {key: Keys.OWNER, name: t('Owner'), width: COL_WIDTH_UNDEFINED},
+    {key: Keys.CREATED, name: t('Created'), width: COL_WIDTH_UNDEFINED},
+  ];
+
+  function handleDelete(dashboard: DashboardListItem) {
+    deleteDashboard(api, organization.slug, dashboard.id)
+      .then(() => {
+        trackAnalytics('dashboards_manage.delete', {
```

Comment:
Not important for now, but it may be interesting to add the view type (table, grid) to the event in case we see different behaviour on either page for actions like delete, duplicate, etc

Reply:
ooo ya good point. I'll make a ticket for that so I remember before we rollout 😄